oneview_redfish_toolkit/api/zone.py

```diff
@@ -87,6 +88,15 @@ def __init__(self,
 
         self._validate()
 
+    def get_zone_name(self, spt_name, resource_id):
+        zone_name = spt_name
+        resource_id_group = resource_id.split("-")
```

Comment:
Here we are not concatenating the enclosure name, but the id, maybe this information will not be friendly user as a Zone name, we could use the enclosure name.